packages/currency/src/index.js

```diff
@@ -159,124 +167,24 @@ const CurrencyFactory = ( currencySetting ) => {
 			}
 			return formatAmount( number );
 		},
+
+		/**
+		 * Format currency from WooCommerce PHP version to WCA JS version.
+		 *
+		 * @param {Object} phpCurrency Currency to format
+		 * @return {Object} Formatted JS currency.
+		 */
+		formatPhpToJs( phpCurrency ) {
+			return {
+				code: phpCurrency.currency_code,
+				symbol: decodeEntities( phpCurrency.symbol ),
+				symbolPosition: phpCurrency.currency_pos,
+				thousandSeparator: phpCurrency.thousand_sep,
+				decimalSeparator: phpCurrency.decimal_sep,
+				precision: phpCurrency.num_decimals,
+			};
+		},
 	};
 };
 
 export default CurrencyFactory;
-
-/**
- * Returns currency data by country/region. Contains code, symbol, position, thousands separator, decimal separator, and precision.
- *
- * Dev Note: When adding new currencies below, the exchange rate array should also be updated in WooCommerce Admin's `business-details.js`.
- *
- * @return {Object} Curreny data.
- */
-export function getCurrencyData() {
```

Comment:
Are we not concerned with backward compatibility in our JS packages?

Reply:
Since we just had a couple breaking change versions, I went ahead and added this back in with a deprecation notice.  My guess is that we're the only ones using it since it's such a limited scope of countries, but always up for erring on the side of safety.